PusherServer/Pusher.cs

```diff
@@ -202,7 +202,21 @@ private IRestResponse ExecuteTrigger(string[] channelNames, string eventName, ob
 
             var request = CreateAuthenticatedRequest(Method.POST, "/events", null, requestBody);
 
+            System.Console.WriteLine("Method: {1}{0}Host: {2}{0}Resource: {3}{0}Parameters:{4}",

```

Comment:
It is. But it's required to work out exactly what the interaction with the HTTP API is. I've created #6 to keep track of that. So, we'll leave this here for now.